addons/website/static/src/snippets/s_image_gallery/001.scss

```diff
@@ -141,47 +141,70 @@
         }
         &.s_image_gallery_indicators_arrows_boxed, &.s_image_gallery_indicators_arrows_rounded {
             .carousel {
-                .carousel-control-prev .fa,
-                .carousel-control-next .fa {
-                    text-shadow: none;
+                .carousel-control-prev,
+                .carousel-control-next {
+                    // Directional icons replaced by oi but keeping fa here, it
+                    // is still used by already dropped snippet
+                    .fa, .oi {
+                        text-shadow: none;
+                    }
                 }
             }
         }
         &.s_image_gallery_indicators_arrows_boxed {
             .carousel {
-                .carousel-control-prev .fa:before {
-                    content: "\f104";
-                    padding-right: 2px;
+                // Need to change font-family: we don't have equalent icon to
+                // fa-angle in odoo-icons
+                .oi {
+                    font-family: "FontAwesome" !important;
```

Comment:
Actually, the simplest would still be to use `oi-chevron` and decreasing the font-size (0.5em maybe?). The visual result will be the same. Could you try that thanks